File: gpao-tex-core/ordre/src/main/java/com/gpro/consulting/ordre/dto/PartieInteresseValue.java
package com.gpro.consulting.ordre.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class PartieInteresseValue {
    Long id;
    Long siteId;
    String reference;
    String matrFiscal;
    String abreviation;
    String adresseLiv;
    String modalitePaiement;
    List<DocumentPiValue> documents;
    List<RepresentantValue> representants;
    String creatorIdentifier;
    String latestUpdaterId;
}

File: gpao-tex-ui/source/main/src/app/services/pi/models/partie-interesse-value.ts
/* tslint:disable */
/* eslint-disable */
import { DocumentPiValue } from '../models/document-pi-value';
import { RepresentantValue } from '../models/representant-value';
export interface PartieInteresseValue {
  abreviation?: string;
  adresseLiv?: string;
  documents?: Array<DocumentPiValue>;
  id: number;
  matrFiscal?: string;
  modalitePaiement?: string;
  reference: string;
  representants?: Array<RepresentantValue>;
  siteId?: number;
}
